metadata-ingestion/src/datahub/ingestion/source/fivetran/config.py

```diff
@@ -167,6 +167,10 @@ class PlatformDetail(ConfigModel):
         description="The database that all assets produced by this connector belong to. "
         "For destinations, this defaults to the fivetran log config's database.",
     )
+    include_schema_in_urn: bool = pydantic.Field(
+        default=True,
+        description="Include schema in the dataset URN. In some cases, the schema is not relevant to the dataset URN and Fivetran sets it to the connector name.",
```

Comment:
```suggestion
        description="Include schema in the dataset URN. In some cases, the schema is not relevant to the dataset URN and Fivetran sets it to the source and destination table names in the connector.",
```

Reply:
I'm not sure if the suggestion improves things, so I'll leave it up to you. It's challenging to make this easily understandable for the end users.